ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -33,6 +33,53 @@ then
     source "${SHARED_DIR}/proxy-conf.sh"
 fi
 
+echo "Collecting cluster data for analysis..."
+set +o errexit
+set +o pipefail
+if [ ! -f /tmp/jq ]; then
+  curl -L https://stedolan.github.io/jq/download/linux64/jq -o /tmp/jq && chmod +x /tmp/jq
+fi
+if ! pip -V; then
+  echo "pip is not installed: installing"
+  if python -c "import sys; assert(sys.version_info >= (3,0))"; then
+    python -m ensurepip --user || easy_install --user 'pip'
+  else
+    echo "python < 3, installing pip<21"
+    python -m ensurepip --user || easy_install --user 'pip<21'
+  fi
+fi
+echo "Installing python modules: json"
+python3 -c "import json" || pip3 install --user pyjson
+PLATFORM="$(oc get infrastructure/cluster -o json|/tmp/jq '.status.platform')" || ""
+TOPOLOGY="$(oc get infrastructure/cluster -o json|/tmp/jq '.status.infrastructureTopology')" || ""
+NETWORKTYPE="$(oc get network.operator cluster -o json|/tmp/jq '.spec.defaultNetwork.type')" || ""
+if [[ "$(oc get network.operator cluster -o json|/tmp/jq '.spec.clusterNetwork[0].cidr')" =~ .*":".*  ]]; then
+  NETWORKSTACK="IPv6" || ""
+else
+  NETWORKSTACK="IPv4" || ""
+fi
+CLOUDREGION="$(oc get node -o json|/tmp/jq '.items[]|.metadata.labels'|grep topology.kubernetes.io/region|cut -d : -f 2| head -1| sed 's/,//g')" || ""
```

Comment:
The cut/head/sed seems very specifically tied to the structure of how the JSON is output by `oc`.  I think jsonpath would be safer/more reliable for these: e.g., `oc get node -o jsonpath='{.items[0].metadata.labels.topology\.kubernetes\.io/region}'`